flintrock/flintrock.py

```diff
@@ -253,6 +253,7 @@ def cli(cli_context, config, provider, debug):
 @cli.command()
 @click.argument('cluster-name')
 @click.option('--num-slaves', type=click.IntRange(min=1), required=True)
+@click.option('--jdk', type=click.IntRange(min=8), default=8)
```

Comment:
I prefer `--jdk-version` or `--java-version` to match the options we have for Spark and Hadoop.

Actually, what's your take on naming this `--jdk-...` vs. `--java-...`? My feeling is that the latter is slightly clearer and more user friendly, but I don't feel strongly about it.